src/pages/health-declaration/health-declaration.html

```diff
@@ -1,36 +1,31 @@
 <ion-header>
+  <mes-header [title]='getTitle()'></mes-header>
   <page-header [deviceLocked]="true"></page-header>
 </ion-header>
 <ion-content padding class="mes-main-content">
   <ion-card class="mes-card">
-    <ion-card-header class="mes-main-card-header mes-breadcrumbs">
-      <span class="mes-header-md mes-breadcrumb-link" navPop>
-        <ion-icon name="arrow-back"></ion-icon>
-        Debrief /
-      </span>
-      <span padding-left class="mes-header-md">Florence Pearson</span>
-      <span float-right>
-        <ion-icon padding-right name="refresh"></ion-icon>
-        <span class="mes-text-sm">Last synced: 8:06</span>
-      </span>
+    <ion-card-header class="mes-main-card-header">
+      <span class="mes-header-lg">Declaration</span>
     </ion-card-header>
     <ion-card-content>
-      <span class="mes-text-md">You must declare any change to your health status since you last applied for a license. It is a criminal
-        offense for you (or anyone else) to make a false statement in order for you to obtain a driving license and can lead to prosecution.
+      <div text-center class="candidate-name">{{getCandidateName() | uppercase}}</div>
+      <div text-center class="candidate-driving-licence">{{getDriverNumber()}}</div>
+      <span class="mes-text-md decleration">You must declare any change to your health status since you last applied for a license.
+        It is a criminal
+        offense for you (or anyone else) to make a false statement in order for you to obtain a driving license and can
+        lead to prosecution.
       </span>
       <div>
         <input type="checkbox" name="health" id="confirm-health" [(ngModel)]="health-confirmation">
-        <label for="confirm-health">I confirm</label>
+        <label for="mes-header-sm">I confirm</label>
```

Comment:
I don't think this `for` label is correct?

Reply:
Thanks, have fixed and now i know why my styling didn't work